src/services/users.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+let users = [];
+
+const getMax = (array) => {
+  if (array.length === 0) {
+    return 1;
+  }
+
+  const maxId = Math.max(...array.map(({ id }) => id));
+
+  return maxId + 1;
+};
+
+const getAllUsers = () => {
```

Comment:
It logical that `all`
```suggestion
const getUsers = () => {
```